flusher.go

```diff
@@ -22,14 +24,14 @@ func (s *Server) Flush(ctx context.Context) {
 	mem := &runtime.MemStats{}
 	runtime.ReadMemStats(mem)
 
-	s.Statsd.Gauge("mem.heap_alloc_bytes", float64(mem.HeapAlloc), nil, 1.0)
-	s.Statsd.Gauge("gc.number", float64(mem.NumGC), nil, 1.0)
-	s.Statsd.Gauge("gc.pause_total_ns", float64(mem.PauseTotalNs), nil, 1.0)
+	span.Add(ssf.Gauge("veneur.mem.heap_alloc_bytes", float32(mem.HeapAlloc), nil),
```

Comment:
Because we use `float32` [in the SSF data types](https://github.com/stripe/veneur/blob/master/ssf/sample.proto#L26): it's not great. Wonder if we can bump the size of those up?